Dockerfile_archive

```diff
@@ -12,28 +12,32 @@ RUN apt-get update --yes \
         iproute2 \
  && apt-get upgrade --yes \
  && apt-get satisfy --yes --no-install-recommends \
+       ## Standard packages
         ca-certificates \
         cargo \
-        dpkg-dev \
         gcc \
         git \
         libclang-dev \
+       #libgpgme-dev \
```

Comment:
Why is this commented out - isn't it necessary for building python3-gpg?

Reply:
This gets cleaned up in another commit that needs to be cherry picked

Reply:
Commit it - please review again